src/utils/fetchWorker.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * @file Runs undici fetch in a worker thread
+ *
+ * Why is this useful?
```

Comment:
Dejligt med en rigtig god forklaring 👏 

der er bare lige den her sætning der er lidt sjov : "The event loop is blocked
 * so much, that logging the actual timings are waiting."

Reply:
Nu har jeg forsøgt at forbedre sætningen